### PR TITLE
cordova osx build fix

### PR DESCRIPTION
**Pull request by daniellacosse**
```
npx gulp build --platform=osx
```

should now work. example:

Screen Shot 2021-11-02 at 5 28 06 PM

cordova added back the `github:` prefixes and updated some things in the `package-lock.json`.

TODOs:

- [x] will regen once i have the client npm stuff in (in draft here: https://github.com/Jigsaw-Code/outline-client/pull/1091, dependent on the server-side migration).
- [x] fix failing check

### CODE REVIEW

**Comment by daniellacosse** [comment]
@fortuna this was what we wanted to fix, right?

**Review by fortuna — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -73,10 +73,10 @@
     "cordova-android": "~9.0.0",
     "cordova-browser": "~6.0.0",
     "cordova-ios": "~6.1.0",
-    "cordova-osx": "~6.0.0",
-    "cordova-plugin-clipboard": "Jigsaw-Code/outline-cordova-plugin-clipboard#v2.0.0",
+    "cordova-osx": "github:Jigsaw-Code/cordova-osx#ce1b7ad",
```
Comment (fortuna) [inline package.json:76]: Let's make sure we pull all the latest changes to our fork before submitting this, and update the reference here.

**Review by fortuna — CHANGES_REQUESTED** [review]

**Comment by daniellacosse** [comment]
@fortuna should we tag a commit or leave it so we can pull in latest master and have our builds track that?

File: src/www/electron.webpack.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ module.exports = makeConfig({
     // @sentry/electron depends on electron code, even though it's never activated
     // in the browser. Webpack still tries to build it, but fails with missing APIs.
     // The IgnorePlugin prevents the compilation of the electron dependency.
-    new webpack.IgnorePlugin(/^electron$/),
+    new webpack.IgnorePlugin({resourceRegExp: /^electron$/}),
```
Comment (fortuna) [inline src/www/electron.webpack.js:57]: Why is this changing? Why is this needed?

Comment (daniellacosse) [reply src/www/electron.webpack.js:57]: i'm not 100% sure 🙁 it was failing here https://github.com/Jigsaw-Code/outline-client/runs/4121937811?check_suite_focus=true#step:7:26 the version must be changed? but i grepped the `package-lock` and didn't see anything obvious. and this passed on the previous PR so... I'm a little confused as well

Comment (daniellacosse) [reply src/www/electron.webpack.js:57]: it must be this minor upgrade? Or one of the adjacent ones?

Screen Shot 2021-11-08 at 2 20 53 PM
 
They shouldn't have changed the interface tho... :/

File: package.json [hunk 3]
```diff
@@ -76,10 +76,10 @@
     "cordova-android": "~9.0.0",
     "cordova-browser": "~6.0.0",
     "cordova-ios": "~6.1.0",
-    "cordova-osx": "~6.0.0",
-    "cordova-plugin-clipboard": "Jigsaw-Code/outline-cordova-plugin-clipboard#v2.0.0",
+    "cordova-osx": "github:Jigsaw-Code/cordova-osx",
```
Comment (fortuna) [inline package.json:79]: I think it's fine to keep like this because package-lock.json will pin the commit.

**Review by fortuna — APPROVED** [review]
